File: contest/b_timing_issue/sol/b_timing_issue.php
<?php
fscanf(STDIN, "%d:%d", $localH, $localM);
fscanf(STDIN, "%d", $ETMinsInEarthMin);
fscanf(STDIN, "%d", $ETMinsInETHour);

$resMins = floor((($localH * $ETMinsInETHour) + $localM) / $ETMinsInEarthMin);
$resHours = floor(($resMins / 60)) % 24;
$resMins = $resMins % 60;
echo($resHours . ":" . $resMins);
?>

File: contest/b_timing_issue/sol/b_timing_issue.cpp
#include <iostream>
#include <cassert>
#include <vector>
#include <unordered_set>
#include <algorithm>
#include <random>
using namespace std;

int main(int argc, char* argv[])
{
   int localH, localM, ETMinsInEarthMin, ETMinsInETHour, resMins, resHours;
   char trash;

   cin >> localH >> trash >> localM >> ETMinsInEarthMin >> ETMinsInETHour;
   resMins = ((localH * ETMinsInETHour) + localM) / ETMinsInEarthMin;

   resHours = (resMins / 60) % 24;
   resMins = resMins % 60;

   cout << resHours << ":" << resMins;

   return 0;
}
